docs/_includes/components/droppable.html

```diff
@@ -0,0 +1,238 @@
+        <div class="m-b-2">
+          <i class="icon icon-upload icon-xxl"></i>
+        </div>
+        <div class="droppable-desktop">
+          <p class="m-b-2">Drag and drop a .CSV file less than 5MB</p>
+          <button class="btn btn-primary">Or select file</button>
+        </div>
+        <div class="droppable-mobile">
+          <p class="m-b-2">Upload .CSV file less than 5MB</p>
+          <button class="btn btn-primary">Select file</button>
+        </div>
+        <div class="droppable-active-cover">
+          <div>
+            <h4 class="m-b-2">Drop file to start upload</h4>
+            <div class="circle circle-sm">
+              <i class="icon icon-add"></i>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-6">
+      <div class="droppable droppable-md droppable-active">
+        <div class="m-b-2">
+          <i class="icon icon-upload icon-xxl"></i>
+        </div>
+        <div class="droppable-desktop">
+          <p class="m-b-2">Drag and drop a .CSV file less than 5MB</p>
+          <button class="btn btn-primary">Or select file</button>
+        </div>
+        <div class="droppable-mobile">
+          <p class="m-b-2">Upload .CSV file less than 5MB</p>
+          <button class="btn btn-primary">Select file</button>
+        </div>
+        <div class="droppable-active-cover">
+          <div>
+            <h4 class="m-b-2">Uploading</h4>
+            <div>
+            <span class='process process-sm'>
+              <span class='process-icon-container'>
+                <span class='process-icon-horizontal'></span>
+                <span class='process-icon-vertical'></span>
+              </span>
+              <svg version='1.1'
+                xmlns='http://www.w3.org/2000/svg'
+                xml:space='preserve'>
+                <circle class='process-circle' cx='50%' cy='50%' r='45%'
+                  fill-opacity='0.0' />
+              </svg>
+            </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+{% highlight html %}
+<div class="droppable droppable-md droppable-active droppable-dropping">
+  ...
+  <div class="droppable-active-cover">
+    <div>
+      ...
+    </div>
+  </div>
+</div>
+<div class="droppable droppable-md droppable-active">
+  ...
+  <div class="droppable-active-cover">
+    <div>
+      ...
+    </div>
+  </div>
+</div>
+{% endhighlight %}
+
+  <h2 id="droppable-mobile">Mobile behaviour</h2>
+  <p>Typically, mobile devices will not have drag and drop capabilities, so we
+    should customise messaging to those users and give an alternative option.</p>
+  <p>We have two classes to help show and hide content in the different scenarios,
+      anything inside <code>.droppable-mobile</code> will only be visible on
+      mobile screen widths.  Anything inside <code>.droppable-desktop</code> will
+      only appear on larger screens.</p>
```

Comment:
These aren't really designed to change styling, they're here to swap out the content.  We do that because mobile users can't drag and drop.  I'll make it clearer, and add the break point info.